tests/data/test_engine/packing-slip.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "name": "c:\\forklift\\warehouse\\warehouse\\sgid\\AGOLPallet.py:AGOLPallet",
```

Comment:
do you want to make these paths generic?

Reply:
Oh yes. Thanks for the reminder...